internal/providers/pluginfw/resources/volume/data_volumes.go

```diff
@@ -35,7 +35,7 @@ type VolumesList struct {
 }
 
 func (d *VolumesDataSource) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
-	resp.TypeName = pluginfwcommon.GetDatabricksStagingName(dataSourceName)
```

Comment:
Can we remove `GetDatabricksStagingName` now?

Reply:
I think it is probably still useful for resources to be migrated in the future. So I'd suggest keeping it for now